packages/gatsby-image/src/index.js

```diff
@@ -100,7 +152,20 @@ const noscriptImg = props => {
     ? `crossorigin="${props.crossOrigin}" `
     : ``
 
-  return `<picture>${srcSetWebp}<img ${width}${height}${sizes}${srcSet}${src}${alt}${title}${crossOrigin}style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/></picture>`
+  let initialSources = ``
```

Comment:
By splitting the logic between "generate source for first image" and "generate source for other images", this section starts to feel really funky. The logic is really the same for all of the sources (generate a source if there is a `media` key, and generate an additional source if there is a `srcSetWebp` key).